pkgs/servers/apache-airflow/default.nix

```diff
@@ -39,12 +39,16 @@ let
 in
 # See note in ./python-package.nix for
 # instructions on manually testing the web UI
-with python.pkgs; (toPythonApplication apache-airflow).overrideAttrs (_:{
+with python.pkgs; (toPythonApplication apache-airflow).overrideAttrs (previousAttrs: {
   # Provide access to airflow's modified python package set
   # for the cases where external scripts need to import
   # airflow modules, though *caveat emptor* because many of
   # these packages will not be built by hydra and many will
   # not work at all due to the unexpected version overrides
   # here.
-  passthru.pythonPackages = python.pkgs;
+  passthru = (previousAttrs.passthru or { }) // {
+    pythonPackages = python.pkgs;
+  };
+    pythonPackages = python.pkgs;
+  };
```

Comment:
Looks like this got duplicated (could have been a bad code suggestion on my part, sorry!)

Reply:
Oops! I should have checked that before pushing.